bitrise.yml

```diff
@@ -58,7 +58,6 @@ workflows:
       - archive-xcode
     after_run:
       - cache-push
-      - deploy-hockeyapp
```

Comment:
Why did you resigned from deploying production build?

Reply:
Because it was not set up for production previously, so it's unneeded.